src/FSharp.Data.Toolbox.Sas/SasFile.fs

```diff
@@ -490,7 +490,7 @@ type SasFile (filename) =
                 let readRow offset len =
                     let rowBytes = decompress meta offset len data
                     seq {
-                    for n = 0 to List.length meta.Columns - 1 do
+                    for n = 0 to Array.length meta.Columns - 1 do
```

Comment:
I think we might get even better performance if this was written using `Array.map` rather than using sequence here.